lib/extensions/virtual_total_mixin.rb

```diff
@@ -108,3 +108,9 @@ def virtual_aggregate_arel(reflection, method_name, column)
     end
   end
 end
+
+module ActiveRecord
```

Comment:
Also, should the module name and filename change since it is no longer in `app/models/mixins`?  Maybe something more like `ArVirtualTotal`?